tslearn/docs/integration_other_software.rst

```diff
@@ -0,0 +1,230 @@
+    >>> to_seglearn_dataset([[[1], [2], [None]], [[1], [4], [3]]])
+    array([array([[1.],
+           [2.]]),
+           array([[1.],
+           [4.],
+           [3.]])], dtype=object)
+
+
+stumpy
+^^^^^^
+
+`stumpy <https://stumpy.readthedocs.io/>`_ is a powerful and scalable Python
+library for computing a Matrix Profile, which can be used for a variety of time
+series data mining tasks.
+:func:`tslearn.utils.to_stumpy_dataset` and :func:`tslearn.utils.from_stumpy_dataset`
+allow users to convert a data set from ``tslearn`` format to ``stumpy`` format
+and vice versa.
+
+.. code-block:: python
+
+    >>> import numpy as np
+    >>> from tslearn.utils import from_stumpy_dataset, to_stumpy_dataset
+    >>> from_stumpy_dataset([np.array([1, 2]), np.array([1, 4, 3])])
+    array([[[ 1.],
+            [ 2.],
+            [nan]],
+
+           [[ 1.],
+            [ 4.],
+            [ 3.]]])
+    >>> to_stumpy_dataset([[[1], [2], [None]], [[1], [4], [3]]])
+    [array([1., 2.]), array([1., 4., 3.])]
+
+
+sktime
+^^^^^^
+
+`sktime <https://alan-turing-institute.github.io/sktime/>`_ is a ``scikit-learn``
+compatible Python toolbox for learning with time series.
+:func:`tslearn.utils.to_sktime_dataset` and :func:`tslearn.utils.from_sktime_dataset`
+allow users to convert a data set from ``tslearn`` format to ``sktime`` format
+and vice versa.
+``pandas`` is a required dependency to use these functions.
+
+.. code-block:: python
+
+    >>> import pandas as pd
+    >>> from tslearn.utils import from_sktime_dataset, to_sktime_dataset
+    >>> df = pd.DataFrame()
+    >>> df["dim_0"] = [pd.Series([1, 2]), pd.Series([1, 4, 3])]
+    >>> from_sktime_dataset(df)
+    array([[[ 1.],
+            [ 2.],
+            [nan]],
+
+           [[ 1.],
+            [ 4.],
+            [ 3.]]])
+    >>> to_sktime_dataset([[[1], [2], [None]], [[1], [4], [3]]]).shape
+    (2, 1)
+
+
+pyflux
+^^^^^^
+
+`pyflux <https://pyflux.readthedocs.io>`_ is a library for time series analysis
+and prediction.
+:func:`tslearn.utils.to_pyflux_dataset` and :func:`tslearn.utils.from_pyflux_dataset`
+allow users to convert a data set from ``tslearn`` format to ``pyflux`` format
+and vice versa.
+``pandas`` is a required dependency to use these functions.
+
+.. code-block:: python
+
+    >>> import pandas as pd
+    >>> from tslearn.utils import from_pyflux_dataset, to_pyflux_dataset
+    >>> df = pd.DataFrame([1, 2], columns=["dim_0"])
+    >>> from_pyflux_dataset(df)
+    array([[[1.],
+            [2.]]])
+    >>> to_pyflux_dataset([[[1], [2]]]).shape
+    (2, 1)
+
+
+tsfresh
+^^^^^^^
+
+`tsfresh <https://tsfresh.readthedocs.io>`_ iis a python package automatically
+calculating a large number of time series characteristics.
+:func:`tslearn.utils.to_tsfresh_dataset` and :func:`tslearn.utils.from_tsfresh_dataset`
+allow users to convert a data set from ``tslearn`` format to ``tsfresh`` format
+and vice versa.
+``pandas`` is a required dependency to use these functions.
```

Comment:
tsfresh is a dependency as well?

Reply:
I do not think so. Why do you think it would?